apps/i18n/common/en_us.json

```diff
@@ -69,6 +69,7 @@
   "animationCategory_tools": "Tools",
   "animationCategory_vehicles": "Vehicles",
   "animationMode": "Animation",
+  "animationPicker_allCategories": "All categories > ",
```

Comment:
I'd recommend we avoid including UI components like `>` in i18n strings; this should probably just be appended to the string